specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -600,8 +600,13 @@
         35,
         36,
         37,
+        38,
         39,
-        41
+        40,
+        41,
+        42,
+        43,
+        44
```

Comment:
Are these just new things?

Reply:
No, these are not new things. We missed some values in the previous PR.
You can refer to our documentation for the complete list [here](https://docs.microsoft.com/azure/azure-maps/weather-services-concepts#weather-icons).